setup.py

```diff
@@ -27,8 +27,10 @@
 http://docs.python.org/2/distutils/sourcedist.html
 
 """
-import os
 from setuptools import setup, find_packages
+from sys import version_info
+import os
```

Comment:
@moggers87 sorry for ne nitpick, but is there any specific reason why you moved the `import os` down? We like to have `import XXX` first, followed by `from XXX import` sorted by alphabet.

If there is no technical reason, could you change this?